apps/swp2clk/boot.js

```diff
@@ -19,17 +19,19 @@
   var DEBUG = false;
   var FILE = "swp2clk.data.json";
 
+  var swipeHandler = (dir) => {
+    log("swipe");
+    log(dir);
+    if (dir === 1) {
+      if (!Bangle.showClock) load();
```

Comment:
Do we need this now? showClock should always be defined after the latest bootloader tweaks

Reply:
If we assume that all apps are up to date, this is not needed.